src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -112,16 +112,16 @@ var HTMLDOMPropertyConfig = {
     label: null,
     lang: null,
     list: MUST_USE_ATTRIBUTE,
-    loop: MUST_USE_PROPERTY | HAS_BOOLEAN_VALUE,
+    loop: MUST_USE_ATTRIBUTE | HAS_BOOLEAN_VALUE,
```

Comment:
Let's do a test here to make sure this still works. It likely wouldn't have been an issue and thrown in IE8 before since this is for HTML5 audio/video.